utils/infer_utils.py

```diff
@@ -71,3 +71,29 @@ def _slice(x):
     exemplar_img.set_shape([batch_size, z_height, z_width, 3])
     return exemplar_img
 
+
+if __name__ == "__main__":
```

Comment:
I am afraid this is not a valid unit test since it makes no use of asserts. 

Minor: 000000.00.crop.x.jpg is nowhere to be found in this repository.